### ISSUE TITLE
Link related components to a Vulcan project to help provide insight

### ISSUE THREAD

**Issue by aaronlippold** [mitre/vulcan#555]

**Comment by ejaronne** [comment]
For example, a user might be building a new STIG based largely on a previous STIG, such as going from Windows 2016 to Windows 2019. While re-accounting for NA and inherently meets is still important wrt the original SRG, jump-starting the process by re-using rules from a previous STIG saves a lot of time!

**Comment by rlakey** [comment]
I don't think Aaron was referencing that type of use case.

The copy component workflow allows someone to copy a component from one project to another to do what you are describing, and jump start the process.

**Comment by ejaronne** [comment]
For example, a user might be building a new STIG based largely on a previous STIG, such as going from Windows 2016 to Windows 2019. While re-accounting for NA and inherently meets is still important wrt the original SRG, jump-starting the process by re-using rules from a previous STIG saves a lot of time!

**Comment by ejaronne** [comment]
Simplifying the ask to get this workflow moving

**Comment by rlakey** [comment]
Seems to be a duplicate of https://github.com/mitre/vulcan/issues/416 then.

To be more specific we don't need to base a new "project" on an official STIG we need to be able to create a "component" from an official STIG.

Once the official STIG is imported then you could copy/duplicate component to a new component to then begin your work.

**Comment by ejaronne** [comment]
The only problem is that the STIG does not have the original SRG content (SRG Requirement, SRG VulDiscussion, SRG Check, SRG Fix), but the new Project still needs full original context.

**Comment by rlakey** [comment]
@ejaronne Just like with import from spreadsheet we choose an SRG to import against in the workflow. We would do the same with importing an official STIG. Import RHEL 8 or whatever, pair it with the GPOS SRG, and any requirements not covered in the STIG are added and set to Not Yet Determined.

I'm using the terms "project" and "component" to be more specific to the constructs in Vulcan. A project is just a collection of components and what you are wanting to import and create is a component and not a project.

**Comment by aaronlippold** [comment]
exactly, we are using Solution of Equations by Interpolation given that each uses a common root - the SRG - to help inform our new solutions on validated ones in the same solution space.
